First Guild Exam Event and Battle Text Updates

John-hash-tech opened the pull request:
This encounter involves four female combatants and some extra event dialog, so it includes a bit more content than my previous branches. I hope to clean up everything up to (but not including) the addition of new free duel opponents.

John-hash-tech committed:
Guild Exam Duel and Event First Pass

I made an effort to fix any obvious typos on the first run through this section of the game. I checked for commas that were not escaped and changed them to semicolons. I also sought out terms in all the related files and that require capitalization and made changes where needed. Some instances of Magic- or Permanent Card were changed to use (Continuous) Spell Card instead.

Changing Great Sword Guardian to Greatsword Guardian was done mostly to suit my preference, though there was already a reference to "Greatsword Warrior" on line 86 of 70007.txt.

I made slight changes to the descriptions of the examiner and Greatsword Guardian in an effort to make them more accurate to the original Japanese.

I also examiner's line 21, since "swordswoman" was gendered in a deliberate way and I though "huntress" would match better.

John-hash-tech committed:
Second Big Pass

Lots more small typo fixes. Line 86 should now fetch the name of Greatsword Guardian instead of using hardcoded text.

Verified that Hunt can special summon monsters above level 4.

John-hash-tech committed:
Update 10018.txt

John-hash-tech committed:
Additional Playtesting

I made an effort to trigger all the usual scenes for each female combatant. I believe there is an unreachable scene that requires special summoning a "self-destructed" monster from the graveyard, which non of the currently-available cards can do, but the text for it looks fine.

John-hash-tech commented:
I am planning to move on to the next round of free duels, which includes Flare, Sister Maria, and Munin. I will return to this PR when there is feedback to address.

Giratena commented on CSV/Language/English/Card/10018.txt:
```diff
@@ -1,17 +1,17 @@
 EU18_PaizuriAnime2,
 大きな剣を巧みに使う頼もしい女剣士。後衛を守るすべに長けていて、彼女への護衛の依頼はいつも絶えない。大きな胸が自慢であり、柔らかい胸を活かしたパイズリが得意。,A reliable swordswoman skilled at utilizing large blades. Because of her proficiency in defending the rear guard; she recieves many requests to act as an escort for young ladies. She is also proud of her large breasts; and is an expert at using their softness to give the best paizuri possible.
-[1].相手ターン開始時に発動する。自分フィールド上のランダムなモンスター1体に、「戦闘ダメージ1割減」（1ターン）を付与する。\n,[1]. At the start of your opponent's turn; grant [90% Damage Reduction] to a random monster on your side of the field for 1 turn.
+[1].相手ターン開始時に発動する。自分フィールド上のランダムなモンスター1体に、「戦闘ダメージ1割減」（1ターン）を付与する。\n,[1]. At the start of your opponent's turn; grant [10% Damage Reduction] to a random monster on your side of the field for 1 turn.
```
Hmm doesn't this effect reduce incoming damage by 90%?

John-hash-tech replied:
Nope. I think the Japanese means something like "1-part discount," but I'm not sure. I confirmed in game that her 1800HP monster could be killed with 2000ATK worth of attacks, though.

Giratena replied:
Hmm ok! If it really is a 10% damage reduction then I have no issues with it, I'm just 90% sure that it reduced almost all damage during my playthrough... You're 100% sure?

John-hash-tech replied:
For some reason, I can't find any explanation for the use of 割減. It looks like 割引 and 減少 are more common for "reduction." 

I do remember seeing this approach to math in Japanese convenience stores, which is explained in this article: http://www.accessj.com/2012/07/reduced-food-labels-in-supermarkets.html

Giratena replied:
Ok, I just tested on my copy of the game to ease my anxities, and it is indeed 10% reduction! Happy for this change to go through!

Giratena commented on CSV/Language/English/Card/50028.txt:
```diff
@@ -1,3 +1,3 @@
 default
-索敵警報,Search Party Alarm
-相手がモンスターの召喚に成功したとき発動できる。自分の手札にレベル5以下のハンタータイプのモンスターがいるなら、そのモンスターを特殊召喚する。それ以外の場合、デッキからハンタータイプのモンスターをランダムに1枚手札に加える。,[1]. If your opponent Summons a monster: Special Summon 1 level 5 or lower Hunter Type monster from your hand. Otherwise; add a random hunter-type monster from your deck to your hand.
+索敵警報,Hunting Party Alarm
```
Hmm I think the original name is fine, "Search Party" is a more common term, and it's also a more faithful translation than the suggestion.

John-hash-tech replied:
Sure. I only suggested the change because (to me) a search party is typically associated with rescue.

Giratena replied:
Hmm, actually while I'm not really feeling hunting alarm, I agree with you there. Search party isn't quite right.

Thinking about it more, the picture of the card is a trip wire, so the meaning seems to be more like an alarm trigger. I think it should be changed thinking about it now so it's a good call out.

What about Hunt Alarm/Tripwire Alarm/Perimeter Alert?

John-hash-tech replied:
Tripwire Alarm and Perimeter Alarm both sound good. I plan to change it to Tripwire Alarm in the next push since that is more clear from the illustration. Perimeter Alarm is the more literal translation, but people might not get that it's like the perimeter of a hunting encampment or whatever.

Giratena replied:
Sounds good! Wonderful change :D

Giratena commented on CSV/Language/English/Card/70007.txt:
```diff
@@ -1,7 +1,7 @@
 試験官,Examiner
 EP07_PaizuriAnime,
-ギルドに所属する中堅マスター。ハンタータイプのカードをメインに使用し、戦士やフクロウのカードをサポートに用いる。\n常日頃から谷間を露出させた格好で出歩いており、人に見られるのがわりと好きな性格。男の子が彼女の胸に見とれていたりすると、両手で寄せ上げてサービスポーズを見せつけたりしてくるので、性癖がゆがむ男の子が後を絶たない。\nランクはC級だが、試験のために実力を抑えている。,A mid-level cardmaster who works at the guild. She mainly uses soldier- and owl-type cards that are built around supporting her hunter-type cards. She enjoys being watched by others; always walking around in  ...
+ギルドに所属する中堅マスター。ハンタータイプのカードをメインに使用し、戦士やフクロウのカードをサポートに用いる。\n常日頃から谷間を露出させた格好で出歩いており、人に見られるのがわりと好きな性格。男の子が彼女の胸に見とれていたりすると、両手で寄せ上げてサービスポーズを見せつけたりしてくるので、性癖がゆがむ男の子が後を絶たない。\nランクはC級だが、試験のために実力を抑えている。,A mid-level cardmaster who works at the guild. She mainly uses Hunter-type monsters; but employs additional Warrior- and owl-type monsters to support them. She enjoys being seen by others; always walking arou ...
```
Nice changes here!

John-hash-tech replied:
Thanks!

Giratena commented on CSV/Language/English/Card/70007.txt:
```diff
@@ -142,29 +142,29 @@ Aランクになるとそれは奇麗な赤色になるんですよ。\nあな
 戦闘中に激しく動いたから、こんな風にゆさゆさ揺れたかもしれないけど…❤その程度ガマンガマン。,Because I've been making large movements during battle; my boobs might have ended up jiggling; just like this...❤ But you'll have to bear with it and hold your feelings in.
 ううっ…❤,Nngh...❤
-え？わざわざ揺らさないでくれって？うそだー、うれしいくせに。,What? "Don't jiggle your boobs like that"? Awh; but you look so happy!
+え？わざわざ揺らさないでくれって？うそだー、うれしいくせに。,What? "Don't jiggle your boobs like that?" Awh; but you look so happy!
```
Maybe single quotes is better here, but the question mark should be **outside** the quote, otherwise it implies the question mark is part of it!

-> What? ‘Don’t jiggle your boobs like that’?

John-hash-tech replied:
Woops, got carried away with this one

Giratena reviewed (changes requested):
Some really nice changes to flow here! Nice job!

I left some comments on some rather suspicious changes that raized my eyebrows a bit. Let me know what you think!

Giratena commented:
Btw just a heads up @John-hash-tech I can't see any changes to this file yet. Did you forget to push? ^^

John-hash-tech commented:
Sorry for the delay! When I formatted my hard drive, I did not evacuate the game folder, so I lost my save data and had to redownload the game + language patch. I was procrastinating starting over because of the holiday weekend and aftereffects of seasonal vaccines, but I am back to work now.

Giratena commented:
> Sorry for the delay! When I formatted my hard drive, I did not evacuate the game folder, so I lost my save data and had to redownload the game + language patch. I was procrastinating starting over because of the holiday weekend and aftereffects of seasonal vaccines, but I am back to work now.

No problem! Hope things are going good and no rush! Just give me a ping when you push and I'll be all over it ^^

John-hash-tech committed:
PR Feedback

Moved a question mark back out of its quotes and changed the name of Hunting Party Alarm again

John-hash-tech commented:
@Giratena We should be ready to roll with this one! I went with "Tripwire Alarm" for that one card.

Giratena reviewed (approved):
Looks awesome! Approved and will merge in ^^
